src/api/app/controllers/webui/users_controller.rb

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
This empty line should be removed.

Beside this, we don't want to move all the actions to this controller?

Reply:
yes, I don't want to do it in this PR, because of the amount of files (views and specs) being changed.

Reply:
Alright, no worries. So removing the empty line is still needed though :smile:

Reply:
done